pygui/__init__.py

```diff
@@ -9,24 +9,22 @@
 from PySide6 import QtCore
 
 try:
+    # If rich is available, use it to improve traceback logs
     from rich.traceback import install
     import shutil
 
     terminal_width = shutil.get_terminal_size((100, 20)).columns - 2
-
     install(width=terminal_width)
 except ModuleNotFoundError:
     pass
 
 from . import pygfx_renderer
 
-__all__ = [
-    "create_element",
-    "render",
-    "use_state",
-]
+
+__all__ = ["create_element", "PyGui"]
 __version__ = version("pygui")
```

Comment:
This doesn't work (out of the box at least) in a pyinstaller distribution

Reply:
This does actually work: it is the same setup as in Arbiter currently.